workspaces/kubernetes-native-dxp/Tiltfile

```diff
@@ -1,3 +1,5 @@
+v1alpha1.extension_repo(name='default', url='https://github.com/gamerson/tilt-extensions')
```

Comment:
this is cool, with one odd thing. Do we need separate functions over at https://github.com/gamerson/tilt-extensions/blob/master/lxc/Tiltfile ? It seems we could use `lxc_webhook` everywhere (with a more generic name) since `darwin_patch` is a pass through when not `uname == Darwin`.

Reply:
done, and I fixed a bug in the darwin_patch